### PR TITLE
Remove `division` related code.

### PR DESCRIPTION
**Pull request by ihongda**

### CODE REVIEW

**Review by sujato — COMMENTED** [review]
Everything looks good.

**Comment by alexander-vo** [comment]
@ihongda Have you checked where the collections that are marked here (`divisions` and `text_divisions`) are used? 
Have you tried to replace them with something, or to make sure that accessing them does not cause errors? 

@robertmusil is it legal to delete migration files like this?
and do we need to create a migration to remove unused collections (`divisions` and `text_divisions`)?

**Comment by ihongda** [comment]
@alexander-vo Alex, I've checked the code and there's no code used the `text_divisions` collection at the time, but I'll check it out again. :)

File: server/server/migrations/migrations/add_divisions_010.py [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-
-        paragraphs = db.create_collection('divisions')
-        paragraphs.add_hash_index(fields=['uid'], unique=True)
-
-    def create_text_divisions_collection(self):
-        db = get_db()
-
-        paragraphs = db.create_collection('text_divisions')
-        paragraphs.add_hash_index(fields=['uid'], unique=True)
-        paragraphs.add_hash_index(fields=['division'], unique=False)
```
Comment (alexander-vo) [inline server/server/migrations/migrations/add_divisions_010.py:20]: In general, I am not sure that we should delete the migration file, as far as I know, it is not quite right :thinking:

Comment (ihongda) [reply server/server/migrations/migrations/add_divisions_010.py:20]: There doesn't seem to be any code that references the two collections at the moment, but we can keep it and delete it later when we're sure it's no longer needed.

Comment (alexander-vo) [reply server/server/migrations/migrations/add_divisions_010.py]: Thanks for bringing him back, he didn't deserve this fate =D

it's not that we need it now, it's just bad practice, maybe later I can say more
personally, I think we need to know what was created when, in what form, for what and who created it + may need additional migration in the future to remove collections that were created here

**Review by alexander-vo — APPROVED** [review]

**Pull request merged**